code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -81,12 +81,14 @@
 				if(!do_after(H, 30))
 					return
 
-				adjustOxyLoss(-(min(getOxyLoss(), 5)))
-				updatehealth()
 				H.visible_message("<span class='danger'>\The [H] performs CPR on \the [src]!</span>")
-				src << "<span class='notice'>You feel a breath of fresh air enter your lungs. It feels good.</span>"
 				H << "<span class='warning'>Repeat at least every 7 seconds.</span>"
 
+				if(istype(H) && health > config.health_threshold_dead)

```

Comment:
Normally I'd agree, but:
A) The code after the `istype(H)` check doesn't reference `H` again.
B) `do_after(H, 30)` is already checking `H` every tick.
C) `H` is referenced before the check.